plugins/python-build/README.md

```diff
@@ -175,6 +175,15 @@ common for performance improvement from this to be in the ballpark of 30%.
 env PYTHON_CONFIGURE_OPTS='--enable-optimizations --with-lto' PYTHON_CFLAGS='-march=native -mtune=native' pyenv install 3.6.0
 ```
 
+If you are using a Mac computer with Apple silicon (e.g. M1), you need to install
+`clang` 15+ before running the command above and choose a recent minor version
+of Python. If you use Homebrew to install `clang` (`brew install llvm`), override
+the compiler to use:
+
+```sh
+env PATH="/opt/homebrew/opt/llvm/bin:$PATH" PYTHON_CONFIGURE_OPTS='--enable-optimizations --with-lto' PYTHON_CFLAGS='-march=native -mtune=native' pyenv install 3.10
```

Comment:
`-march`? What happens to automatically deducing architecture from `arch`? Will this work when compiling for either `arm64` or `amd64`?

Reply:
This is from the original command in the README (L175, just above my change). I copied the command as is and just added Homebrew's LLVM.

Reply:
The original command is for Linux where there typically isn't an issue of two native architectures like in MacOS.

Reply:
I can try another command if you'd like?